integration_tests/tests/utils/miner.rs

```diff
@@ -70,225 +41,42 @@ pub fn register_miner_process(world: &mut TariWorld, miner_name: String, base_no
         wallet_name,
         mine_until_height: 100_000,
     };
-    world.miners.insert(miner_name, miner);
-}
-
-pub async fn mine_blocks(world: &mut TariWorld, miner_name: String, num_blocks: u64) {
```

Comment:
I think that for the time being we will need this code

Reply:
I put everything back in. And I'm using the new approach for the general mining.